app/helpers/maintenance_tasks/tasks_helper.rb

```diff
@@ -182,5 +182,18 @@ def datetime_field_help_text
         class: "content is-small",
       )
     end
+
+    # Mask sensitive information from the arguments of a task.
+    #
+    # @param run [Run] the Run that contains the arguments to be masked.
+    def mask_arguments(run)
```

Comment:
What do you think about having the `Run` handle this instead? That would allow us to memoize the parameter filter.

```ruby
module MaintenanceTasks
  class Run < ApplicationRecord
    ...
    def masked_arguments
      argument_filter.filter(arguments)
    end

    private

    def argument_filter
      @argument_filter ||= ParameterFilter.new(Rails.application.config.filter_parameters + task.masked_attributes)
    end
```